website/README.md

```diff
@@ -98,31 +101,25 @@ Stop the postgresql service using
     pg_ctl -D /usr/local/var/postgres stop  # if not installed using homebrew
     
 ### Linux
-**_NOTE_**:
 
-Currently the environment does not run well on Windows Bash / WSL ( Windows Subsystem for Linux ).
-There are too many issues with line terminators and other environment inconsistencies.
-
-The best option is to run "bare metal" Linux or dual boot. You can run Linux in a VM, but performance will suffer, buyer beware.
-
-#### Direnv
+#### Pip
+Install pip on your local machine in order to setup a virtual environment. [Setup](https://pip.pypa.io/en/stable/installation/) 
+Then install pipenv to create the virtual environment shell.
 
-Install direnv on your local machine, and set it up so it works
-in your shell. These are the instructions for the (default) bash shell. If
-you're using a different shell, you probably know where to configure it for
-yours or the check the [direnv setup page](https://direnv.net/docs/hook.html) for your shell:
+    pip install --user pipenv
```

Comment:
Adding separate instructions for Linux users to follow through with pip and pipenv